packages/deployments/contracts/src/cli/ownership/transferOwnership.ts

```diff
@@ -205,15 +220,31 @@ export const transferOwnership = async () => {
         // handle separately
         continue;
       }
-      await handleOwnership((deployments as Deployment).name, (deployments as Deployment).contract, provider, domain);
+      if ((deployments as Deployment).name.includes("RelayerProxy")) {
+        await handleOwnership(
```

Comment:
We need a mapping to config which contract should be transferred to `admin` /`dao`?